addon/helpers/multi-select.js

```diff
@@ -1,10 +1,25 @@
 import Ember from 'ember'
 const {Helper, isArray} = Ember
-const {helper} = Helper
 
 // TODO Better repo location?
 export function multiSelect ([selectedItems]) {
-  return isArray(selectedItems) && selectedItems === 0
+  return isArray(selectedItems) && selectedItems.length === 0
```

Comment:
this would also be enabled if just one item is selected. So its basically covering if *any* item/s are selected.